### ISSUE TITLE
Add support for Editable Children feature

### ISSUE THREAD

**Issue by magian1127** [Cat-Lips/GodotSharp.SourceGenerators#15]
![image](https://user-images.githubusercontent.com/14800320/188915289-965eeb1c-03ec-4dcf-b88c-b569e2e258cf.png)

It should be caused by this node.
![image](https://user-images.githubusercontent.com/14800320/188915891-ed55f1a2-3d91-41d4-8f68-a03f521c1166.png)

[testS.zip](https://github.com/Cat-Lips/GodotSharp.SourceGenerators/files/9507460/testS.zip)

**Comment by Cat-Lips** [comment]
Hi.  Thanks for reporting this.  I have learned something new!

I will need to provide an update for Editable Children to be automatically detected, but for now you can add `traverseInstancedScenes: true` to the SceneTree declaration as a workaround:
```
[SceneTree(traverseInstancedScenes: true)]
public partial class ZhTest : Node2D
{
    ...
}
```

**Comment by Cat-Lips** [comment]
I've released 1.3.1 for current fixes.  I'll resolve this issue in 1.4.0 by removing the flag and always including the child hierarchy.

**Comment by Cat-Lips** [comment]
New pre-release package added to nuget.
You can remove the traverseInstancedScenes workaround.

Good luck with your game :)
